js/hitbtc2.js

```diff
@@ -1474,6 +1474,10 @@ module.exports = class hitbtc2 extends hitbtc {
             if ((code === 503) || (code === 504)) {
                 throw new ExchangeNotAvailable (feedback);
             }
+            // {"code":429,"message":"Too many requests","description":"Too many requests"}
+            if ((code === 429)) {
```

Comment:
Can you move it to `exceptions` in `describe()` ? )

Reply:
Actually, nevermind, we can just `return;` to fall back to the default handler that already does handle 429 as `DDOSProtection`.